com.unity.render-pipelines.universal/Tests/Editor/EditorTests.cs

```diff
@@ -4,6 +4,7 @@
 using UnityEngine.Rendering;
 using UnityEngine.Rendering.Universal;
 using UnityEditor.Rendering.Universal.Internal;
+using UnityEngine.Experimental.Rendering.Universal;
```

Comment:
Which API is requiring experimental namespace?

Reply:
Renderer2D